Formula/docker-slim.rb

```diff
@@ -14,12 +15,14 @@ class DockerSlim < Formula
   depends_on "go" => :build
 
   def install
-    ENV["CGO_ENABLED"] = "0"
-    ldflags = "-s -w -X github.com/docker-slim/docker-slim/pkg/version.appVersionTag=#{version}"
-    system "go", "build", "-trimpath", "-ldflags=#{ldflags}", "-o", bin/name, "./cmd/docker-slim"
+    system "make", "build"
+    bin.install "dist_mac/docker-slim"
+    chmod 0555, "dist_mac/docker-slim-sensor"
+    bin.install "dist_mac/docker-slim-sensor"
   end
 
   test do
     assert_match version.to_s, shell_output("#{bin}/docker-slim --version")
+    system "test", "-x", bin/"docker-slim-sensor"
```

Comment:
```suggestion
    assert_match version.to_s, shell_output("#{bin}/docker-slim-sensor --version")
```

Reply:
docker-slim-sensorはLinux環境用に構築する必要があるため、macOSで実行できません。

Reply:
docker-slim-sensor needs to be built for the Linux environment, so it cannot run on the macOS.

Reply:
if it can't run on all of the systems that homebrew supports it's not a good fit for homebrew core and should be kept in a separate tap.

Reply:
docker-slim-sensor needs to be built for the Linux environment, and it is the correct thing. docker-slim-sensor is used in the Linux container so that it can run on both macOS and GNU/Linux.